drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -90,6 +90,7 @@ insertRowAbove=Insert row above
 insertRowBelow=Insert row below
 deleteRow=Delete row
 duplicateRow=Duplicate row
+runSingleScenario=Run single scenario
```

Comment:
Only one thing: can we rename it to just _Run scenario_? It's simpler and I think sufficient.